piecrust/src/vm.rs

```diff
@@ -60,6 +98,17 @@ impl VM {
         self.host_queries.insert(name, query);
     }
 
+    /// Spawn a [`Session`] with the given commit as the `base`.
+    ///
+    /// Sessions spawned with a commit as a base are running modifications to
+    /// said commit. For sessions with no commit as base use
```

Comment:
For sessions with no commit as base, use ...

Reply:
Line below is "[`genesis_session`]"